package.json

```diff
@@ -3,7 +3,11 @@
   "description": "Hoodie Client Core",
   "main": "index.js",
   "scripts": {
-    "test": "standard && npm run -s test:node | tap-spec",
+    "build": "npm-run-all -p build:*",
```

Comment:
Not sure where I'd use "postbuild" here.
`npm-run-all` is the most straightforward way to run the tasks in parallel and also appears to be future proof should we ever decide to add more builds (e.g. hoodie.js with external lodash build).